components/Storyblok/SbLocalFooter.tsx

```diff
@@ -28,83 +27,81 @@ export type SbLocalFooterProps = {
 };
 
 const styles = {
-  linkGroup: cnb('list-none m-0 p-0 [&_a]:leading-snug [&_a]:font-normal [&_a]:text-digital-red [&_a:hover]:text-black [&_a:focus]:text-black'),
-  linkGroupHeading: cnb('text-20 font-serif tracking-normal'),
-}; 
+  linkGroup: 'list-none m-0 p-0 [&_a]:leading-snug [&_a]:font-normal [&_a:hover]:text-black [&_a:focus]:text-black',
+  linkGroupHeading: 'text-20',
+};
 
 export const SbLocalFooter = (props: SbLocalFooterProps) => (
-  <div {...storyblokEditable(props.blok)} className="pb-38 pt-34 md:pb-72 md:pt-58 bg-white">
-    <CenteredContainer>
-      <div className="pb-32 md:pb-45">
```

Comment:
When Giving was built, the modular spacing system was not quite there yet. I've replaced things like `pb-32 md:pb-45` with `rs-pb-3` for example - at 2XL it's slightly larger but not noticeable to most people, and I'd rather use the new system. Let me know what you think 😄